docs/_docs/getting-started.md

```diff
@@ -153,6 +141,36 @@ $ tributors update zenodo
 $ tributors update codemeta
 ```
 
+If the client finds more than one orcid identifier for a name, you'll be prompted
+to run in `--interactive` mode. Running in interactive mode will allow you
+to choose a number for each one:
+
+```bash
+$ tributors update zenodo --interactive
+INFO:    zenodo:Updating .zenodo.json
+INFO:    zenodo:Updating .tributors cache from .zenodo.json
+
+Meyer, Kyle
+======================================================
+[1]
+  Name: Meyer, Kyle
+  Orcid: 0000-0002-1933-2908
+  Institutions: University of California Davis, University of Michigan, University of Texas at Austin
+
+[2]
+  Name: Meyer, Kyle
+  Orcid: 0000-0001-8632-4425
+  Institutions: Kroger Co, Northeastern Ohio Medical University, University of Toledo, University of Toledo Medical Center
+
+[3]
+  Name: Meyer, Kyle
+  Orcid: 0000-0001-8846-7411
+  Institutions: University of Auckland, University of Oregon, University of Wisconsin Milwaukee
```

Comment:
nice! what about emails?  if known - could it be checked or at least displayed here? I seems to have one listed:
```
$> grep email myorcid.json 
    "verified-email": true,
    "verified-primary-email": true
    "emails": {
      "email": [
          "email": "debian@onerussian.com",
      "path": "/0000-0003-3456-2493/email"
```
although it seems not everyone does:
```
$> curl --silent -X GET --header 'Accept: application/json' 'https://pub.orcid.org/v3.0/0000-0002-4387-3819' | jq . | grep email      
    "verified-email": true,
    "verified-primary-email": true
    "emails": {
      "email": [],
      "path": "/0000-0002-4387-3819/email"
```
;)

Reply:
If the email exists it will be printed. If I can remember correctly, the majority of users don't make their email public.